src/admin-views/editor/fieldset/price.php

```diff
@@ -4,19 +4,25 @@
 	$post_id = get_the_ID();
 }
 $validation_attrs = [
-	'data-validation-error="' . esc_attr( sprintf( _x( '%s price must be greater than zero.', 'ticket price validation error', 'event-tickets' ), tribe_get_ticket_label_singular( 'ticket_price_validation_error' ) ) ) . '"'
+	'data-validation-error="' . esc_attr( sprintf(
+		// Translators: %s: singular version of the Ticket label.
+		_x( '%s price must be greater than zero.', 'ticket price validation error', 'event-tickets' ),
+		tribe_get_ticket_label_singular( 'ticket_price_validation_error' )
+	) ) . '"'
```

Comment:
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 4113586095854219768f19de81a0c66d</sub>